front/pages/w/[wId]/spaces/[spaceId]/index.tsx

```diff
@@ -45,12 +45,14 @@ export const getServerSideProps = withDefaultUserAuthRequirements<
   }
 
   const canWriteInSpace = space.canWrite(auth);
+  const isBuilder = auth.isBuilder();
```

Comment:
Why is that not covered by the canWrite above? Shouldn't we fix this one instead?

Reply:
this is needed for the apps, as `canWriteInSpace` logic's doesn't work for apps / tools specifically
if i'm a user and i'm in my private space, I can technically write in space, but i should not be able to create a dust app

Reply:
Copy that

Reply:
we actually need a combination of both so that builder NOT in space A can't write in space A

Reply:
Copy that